modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -874,19 +871,37 @@ private void copyInBuffer(long absPtr, ByteBuffer tmpBuf) {
     }
 
     /**
-     * Get current prartition generation tag.
+     * Get current partition generation tag.
      *
      * @param seg Segment.
-     * @param fullId Full page id.
+     * @param fullPageId Full page id.
      * @return Current partition generation tag.
      */
-    private int generationTag(Segment seg, FullPageId fullId) {
+    private int generationTag(Segment seg, FullPageId fullPageId) {
         return seg.partGeneration(
-                fullId.groupId(),
-                partitionId(fullId.pageId())
+                fullPageId.groupId(),
+                partitionId(fullPageId.pageId())
         );
     }
 
+    /**
+     * Get current partition generation tag.
+     *
+     * @param fullPageId Full page id.
+     * @return Current partition generation tag.
+     */
+    public int generationTag(FullPageId fullPageId) {
```

Comment:
Since we are writing page **PartitionMetaIo** out of **PersistentPageMemory** and directly to a file, does it seem like we have to write the same generation for it, or can we omit that?

Reply:
Got rid of the method and associated code.